src/views/ItemView.test.js

```diff
@@ -6,14 +6,22 @@ import { waitFor, screen } from '@folio/jest-config-stripes/testing-library/reac
 
 import '../../test/jest/__mock__';
 
-import { StripesContext, ModuleHierarchyProvider } from '@folio/stripes/core';
+import {
+  StripesContext,
+  ModuleHierarchyProvider,
+  checkIfUserInCentralTenant,
+} from '@folio/stripes/core';
 import { renderWithIntl, translationsProperties } from '../../test/jest/helpers';
 
 import ItemView from './ItemView';
 
 jest.mock('../Item/ViewItem/ItemAcquisition', () => ({
   ItemAcquisition: jest.fn(() => 'ItemAcquisition'),
 }));
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  checkIfUserInCentralTenant: jest.fn(),
+}));
```

Comment:
Please don't override the existing `stripes/core` mock, you can remove `<StripesContext.Provider value={stripesStub}>` from this file and mock `checkIfUserInCentralTenant` in `stripesCore.mock.js`. So this file will only contain one version of `stripes` mock.

Reply:
good point, fixed